build-tools/scripts/RunTests.targets

```diff
@@ -83,5 +83,18 @@
         Properties="AotAssemblies=True"
     />
   </Target>
-  <Target Name="RunAllTests" DependsOnTargets="RunNUnitTests;RunJavaInteropTests;RunApkTests" />
+  <ItemGroup>
+    <_RunTestTarget Include="RunNUnitTests" />
+    <_RunTestTarget Include="RunJavaInteropTests" />
+    <_RunTestTarget Include="RunApkTests" />
+  </ItemGroup>
+  <Target Name="RunAllTests"
+      Inputs="@(_RunTestTarget)"
+      Outputs="%(Identity).__does_not_exist">
```

Comment:
I want to batch the target.

If there were no inputs or outputs, instead of passing one item to the `<MSBuild/>` task, we'd instead pass the *entire item group* to [`MSBuild.Targets`](https://msdn.microsoft.com/en-us/library/z7f65y0d.aspx).

However, there's also a `MSBuild.RunEachTargetSeparately` property, which is perhaps better?

> If `true`, the MSBuild task invokes each target in the list passed to MSBuild one at a time, instead of at the same time. Setting this parameter to `true` guarantees that subsequent targets are invoked even if previously invoked targets failed. Otherwise, a build error would stop invocation of all subsequent targets. Default is `false`.